tests/modules/test_encoders.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+import tempfile
+import unittest
+
+from mmf.modules import encoders
+from tests.test_utils import skip_if_no_network
+from torch import nn
+
+
+class TestEncoders(unittest.TestCase):
+    def _test_init(self, cls, **params):
```

Comment:
We can use proxy to enable running these tests internally as well. You can call  `test_utils.setup_proxy()` here (or better once in a `setUp` method). Then we can remove the `@skip_if_no_network`.